third-party/realdds/include/realdds/topics/dds-topic-names.h

```diff
@@ -21,5 +24,179 @@ constexpr char const * NOTIFICATION_TOPIC_NAME = "/notification";
+            extern std::string const stream_name;
+        }
+    }
+    namespace query_option {
+        extern std::string const id;
+        namespace key {
+            using control::set_option::key::option_name;
+            using control::set_option::key::stream_name;
+        }
+    }
+    namespace query_options {
+        using notification::query_options::id;
+        namespace key {
+            using control::set_option::key::option_name;
+            using control::set_option::key::stream_name;
+            extern std::string const sensor_name;
+        }
+    }
+    namespace open_streams {
+        extern std::string const id;
+        namespace key {
+            extern std::string const stream_profiles;
+            extern std::string const reset;
+            extern std::string const commit;
+        }
+    }
+    namespace hwm {
+        extern std::string const id;
+        namespace key {
+            extern std::string const opcode;
+            extern std::string const param1;
+            extern std::string const param2;
+            extern std::string const param3;
+            extern std::string const param4;
+            extern std::string const data;
+            extern std::string const build_command;
+        }
+    }
+    namespace hw_reset {
+        extern std::string const id;
+    }
+}
+
+namespace reply {
+    namespace key {
+        using control::key::id;
+        extern std::string const sample;
+        extern std::string const control;
+        extern std::string const status;
+        extern std::string const explanation;
+    }
+    namespace status {
+        extern std::string const ok;
+    }
+    namespace set_option {
+        using control::set_option::id;
+        namespace key {
+            using control::set_option::key::value;
+        }
+    }
+    namespace query_option {
+        using control::query_option::id;
+        namespace key {
+            using control::set_option::key::value;
+        }
+    }
+    namespace query_options {
+        using control::query_options::id;
+        namespace key {
+            using notification::query_options::key::option_values;
+        }
+    }
+    namespace open_streams {
+        using control::open_streams::id;
+        namespace key {
+            using control::open_streams::key::stream_profiles;
+        }
+    }
+    namespace hwm {
+        using control::hwm::id;
+        namespace key {
+            using control::hwm::key::data;
+        }
+    }
+}
+
+namespace metadata {
+    namespace key {
+        extern std::string const stream_name;
+        extern std::string const header;
+        extern std::string const metadata;
+    }
+    namespace header {
+        namespace key {
+            extern std::string const frame_number;
+            extern std::string const timestamp;
+            extern std::string const timestamp_domain;
+            extern std::string const depth_units;
+        }
+    }
+    namespace metadata {
```

Comment:
Needed? Looks like copy paste leftover

Reply:
I think it's there because there actually is a `metadata` key inside the `metadata` topic messages:
```
{
  "stream-name": "Color",
  "header": {
    ...
  },
  "metadata" : {
    ...
  }
}
```
It's just that currently there are no predefined keys inside. It's nice to have this with intellisense.